src/app/App.tsx

```diff
@@ -171,71 +202,88 @@ export function App(): React.JSX.Element {
-        <div className="workflow-activity-container">
+          <div className="workflow-activity-container">
             <h3 className="activity-heading">Workflow Activity</h3>
             <WorkflowActivityList />
-        </div>
-        <div className="snippets-container">
-          <div className="snippets-header">
-            <div className="snippets-subheader">
-              <h2 className="snippets-heading">Saved Snippets</h2>
+          </div>
+          <div className="snippets-container">
+            <div className="snippets-header">
+              <div className="snippets-subheader">
+                <h2 className="snippets-heading">Saved Snippets</h2>
               <button className="refresh-btn" onClick={refreshSnippetList}>Refresh Snippet List
-              </button>
+                </button>
               <button className="deselect-btn" onClick={handleDeSelect}>DESELECT
-              </button>
+                </button>
               <DeleteAssetButton assetID={((selectedIndex < array.length && selectedIndex!=-1) ? array[selectedIndex].id : "" )} selectedIndex={selectedIndex} setArray={setArray}/>
-            </div>
+              </div>
 
-            <div className="snippets-grid">
+              <div className="snippets-grid">
             {
               array.filter(item => searchTerm === '' || item.name.includes(searchResult)).length > 0 ?
               array.filter(item => searchTerm === '' || item.name.includes(searchResult)).map((item: LocalAsset, index) => (
-                <div
-                  onKeyDown={handleKeyPress}
-                  tabIndex={0}
-                  key={index}
-                  className="snippet-item"
-                  style={{
+                      <div
+                        onKeyDown={handleKeyPress}
+                        tabIndex={0}
+                        key={index}
+                        className="snippet-item"
+                        style={{
                     backgroundColor: selectedIndex == index ? 'lightblue' : 'white', // Add background color based on selection
-                  }}
-                  onClick={() => handleSelect(index)}
-                >
-                  <div>
-                    <h3>{item.name}</h3>
-                    <div className="snippet-details">
-                      <p>{item.id}</p>
-                      <p>{item.classification}</p>
-                    </div>
-                  </div>
-                </div>
-              ))
+                        }}
+                        onClick={() => handleSelect(index)}
+                      >
+                        <div>
+                          <h3>{item.name}</h3>
+                          <div className="snippet-details">
+                            <p>{item.id}</p>
+                            <p>{item.classification}</p>
+                          </div>
+                        </div>
+                      </div>
+                    ))
```

Comment:
Can you please remove these unnecessary spaces?

Reply:
Sure, thank you for the critical review. I will remove the unnecessary spaces.

As above spaces is because auto formatting tool I use daily for better reading of code.

I would prospose, it is good to use a standard formatting protocol for the project -
1. We can use prettieir or other formatting tools (I observed many developers use it in project)
2. If we go with optiob 1. we should add in docs/readme to make sure other contributors use it too.

Thanks.

Reply:
Thanks for the suggestion, Vishal! We'll look into it!

Reply:
Done